logger/index.js

```diff
@@ -1,71 +1,62 @@
-var bunyan = require('bunyan'),
-	fs = require('fs')
-
+var winston = require('winston'),
+	fs = require('fs'),
+	dateFormat = require('dateformat'),
+	util = require('util'),
+	path = require('path'),
+	pad = require('pad');
+	
 exports.createLogger = createLogger
 exports.getInstance = getInstance
 
-var instance
-// Rotate the file every day
-var logType = 'rotating-file' 
-var logPeriod = '1d'
-// By default don't show extra logs, intensive and un-needed.
-var isOnRoids = false
+var instance;
 
 // Public
 function createLogger (name, defaultPath, opts) {
-	var level = opts.verbose ? opts.verbose : opts.sVerbose
-	var preferedPath = opts.path ? opts.path : defaultPath // If we're not in-memory use the same directory for logs.
-	
-	var validatedLvl = validateLevel(level)
-	if (!validatedLvl)
-		return undefined
-	
-	createLogDirectoryIfNeeded(preferedPath)
-	var fullPath = preferedPath + '/dynaLog/' + getLogFileName(name)
-	isOnRoids = opts.sVerbose ? true : false // Do we want the serializers to show extra details?
-	
-	if (!instance)
-	{
-		instance = bunyan.createLogger({
-				name: name,
-				serializers: SERIALIZERS,
-				streams: [{
-					type: logType,
-					level: validatedLvl,
-					path: fullPath,
-					period: logPeriod
-				}]
-			}
-		)
+	if (!opts.verbose) {
+		instance = {
+			info:function() {},
+			debug:function() {},
+			error:function() {},
+			warn:function() {},
+		}
 		
-		console.log("Writing logs to - " + fullPath)
+		return instance;
 	}
 	
-	return instance
-}
+	var dataDirectory = opts.path ? opts.path : defaultPath // If we're not in-memory use the same directory for logs.
+	var logsDirectory = getValidLogDirectory(dataDirectory);
+	var fullPath = path.join(logsDirectory, getLogFileName(name));
 
-// Public
-function getInstance () {
+	instance = new (winston.Logger)({
+		transports: [
+			new (winston.transports.File)({
+				timestamp: function() {
+					return Date.now();
+				},
+				formatter: function(options) {
+					var formattedTime = dateFormat(options.timestamp(), "yyyy-mm-dd h:MM:ss");
+					
+					var msg = util.format("%s %s %s", 
+						pad(formattedTime, 20),
+						pad(options.level.toUpperCase(), 6),
+						options.message);
+					
+					return msg
+				},
+				filename: fullPath,
+				json: false,
+				level: "debug"
```

Comment:
We don't always use logger in our "dynalite". But, when we use it we put "debug" to see everything. 

Look at the "if (!opts.verbose) {" in the beginning of the method.